esi_leap/tests/api/controllers/v1/test_utils.py

```diff
@@ -410,57 +208,77 @@ def test_get_offer_name_available(self,
              'status': statuses.AVAILABLE}
         )
 
-    @mock.patch('esi_leap.api.controllers.v1.utils.'
-                'lease_authorize_management')
     @mock.patch('oslo_utils.uuidutils.is_uuid_like')
     @mock.patch('esi_leap.objects.lease.Lease.get')
-    def test_get_lease_authorized_uuid(self,
-                                       mock_lease_get,
-                                       mock_is_uuid_like,
-                                       mock_authorize):
+    def test_get_lease_uuid(self, mock_lease_get, mock_is_uuid_like):
+
+        mock_is_uuid_like.return_value = True
+        mock_lease_get.return_value = test_lease
+
+        res = utils.get_lease(test_lease.uuid)
+
+        self.assertEqual(res, test_lease)
+
+        mock_is_uuid_like.assert_called_once_with(test_lease.uuid)
+        mock_lease_get.assert_called_once_with(test_lease.uuid)
+
+    @mock.patch('oslo_utils.uuidutils.is_uuid_like')
+    @mock.patch('esi_leap.objects.lease.Lease.get')
+    def test_get_lease_uuid_available(self,
+                                      mock_lease_get,
+                                      mock_is_uuid_like):
 
         mock_is_uuid_like.return_value = True
         mock_lease_get.return_value = test_lease
 
-        res = utils.get_lease_authorized(test_lease.uuid,
-                                         lessee_ctx.to_policy_values())
+        res = utils.get_lease(test_lease.uuid, [statuses.CREATED])
```

Comment:
Ah, leases don't have an available status; only offers do. It's because leases are never claimed or used by others; they're simply created, fulfilled, and then expired.